src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -224,22 +229,21 @@ const SatelliteManifestPanel: FunctionComponent<SatelliteManifestPanelProps> = (
     const results = [
       {
         title: 'Export',
+        disabled: false,
         onClick: () => {
           exportManifest(uuid, name);
         }
-      }
-    ];
-    if (user.canWriteManifests) {
-      results.push({
+      },
+      {
+        disabled: true,
         title: 'Delete',
         onClick: () => {
-          openDeleteConfirmationModal(uuid);
+          handleOnClickforKebab();
```

Comment:
We still need to call the `openDeleteConfirmationModal` function to delete a manifest when this button is enabled